split_file.py

```diff
@@ -1,19 +1,46 @@
 import csv
+import sys
 
-input_file_name = './test.csv'
-output_neural_in = './neural_in.csv'
-output_neural_out = './neural_out.csv'
+def split_file(num_blocks, input_file_name, output_neural_in, output_neural_out):
+    with open(input_file_name, 'r') as csv_in_file:
```

Comment:
We can flatten out this nesting a bit: https://docs.python.org/3.4/reference/compound_stmts.html#with

Reply:
Did not know you could do that. neat-o